src/client/app/admin/authentication/user-state.service.ts

```diff
@@ -41,4 +45,11 @@ export class UserStateService {
 	public clearUser() {
 		this.user.clearUser();
 	}
+
+	private routeError(err: any): void {
+		// console.log('RouteError');
+		if (err.status === 401) {
+			this.clearUser();
```

Comment:
I think the difference is between 401 (which is supposed to be for authentication) vs. 403 (which is supposed to be used for authorization). 403s should be used for unauthorized resources, whereas 401s would be returned if the user isn't authenticated or authenticated improperly (e.g., lost session). http://stackoverflow.com/a/6937030